packages/oui-loader/_variables.less

```diff
@@ -1,9 +1,16 @@
FxdFFBCnZ0NEF2ZDRBdmQ4QXZ1QUF2T0FBdmQ0QXZ1QUF2dDhBdnVBQXZkOEF2T0FBdmQ4QXZ1QUF2ZUE2M0ZFakFBQUEvblJTVGxNZnpTUkwKSnFUSG5sRGZXRzlkbUkydjNkdkxGSWRrTFZlT21yTEExKzd6SUJwemNSZHNYeE5obktiaUFPWWNIbjRwT0NOV1NUMTRNSUZaUm5rTgpYbjFwbTZHam9CdkVxb2k4cDVDcHJBbWRCK0hPNWZJWlFTSmJUeXlEYm1wRVgyaUtDMmVYbExaak03ZXUxdGphQk5uVDRkVHA2eFlYClBoaUFSMHhUS0dZM2hTczFranFGb21NeHM4WGYwY25HME9UU3VRK3c1K2dkSENGdGV5ZENFRlIzUXhhTFlqdVprMlVwa1MrZnVuYTkKWU9DMXFNRFYzZXI3RWtnTVVqUjFNbHhjYXdoT1ZYcEtPWlhqQWNMNDgvMytEallWUEJHVmxwMm1xN3drdU1qUDZlZnQ4UFgzOU84WQpjaitKZEZXWDN0UzBCdXp2dE1HbXQ3N2UrdlhRNndzKzhySUFBQVVpU1VSQlZEakxOZEI1UE5SNUhBYndyMk9haVVaam5NTnNpQnl6Cm8xd2xaTjI4M1BkVnhMcXZ6YldiR3NrOUlRekpsU1ZIUXBHYmRMQXBaZGF4YlZsS3RURmxkV3EzOXFyZFY3dWY3Mi8wdk9iUDl6elAKOS9ORHBIeVNqbzZqczcxTDdPWUtwZDYveWpzWC8xYTJLOWxRVEtkN0huL3pKbzU1UUhyd2ZkRkdGaHRabmJmSnR5YXRVNlZmSTJQSwpGeS8wMjlrUnNyUXNjU0ErUGtHYVJ1dmhvZk51WktBazBwOTlJQ3VVQ2lJZkFweXlLM3lFS3hPVDQ1anhDWU8wdFNJZVd0QUdhWlZQCnV1Zm9mSitvQktnOFpWZnlpQzd5OC9IRUFaaVdCdWZoaGR3c1FkcFl3VHBSV2RBYjAzbEJtWmlHNVdRbUhsNHI2dUd6a1Raalhlb0kKSzRXd2NBT2REb1g0RmhxdHFDZUp2NFFZVktITXh3ZTV4Q3I5SHRPNXFJeVg2VkFZeDB3Z0hJdGZpYWl2cWEwTVN6ZDgwRDFuKzFpNApwbnl4MzY2d0dKK0NYemhJT040UW9rN09nOVRXSnR0WWt4enRZVnNJOFNuSmNUQmNWTVJpOGIxT3BxTFhFWlB6NFZRR1l3R2tUdC85CldLWGVoM2lhTGxLYUdCY3ZUWHVmNU1IbmVTMmxMS09JanJNUmsrSFVWc3NGc3BYMVBXZVhpb0lZZ0J2b25tVUQ4UWx3TUl2SFl5OVYKRHYyQ2NwdzZjczVHaEllM1dtcVRyZkQ0T2l4TlppWklZOGRtcHp4Ky9GYUFuQlFWTWYxcG5zckFzczlGS1hKUnVhVFlzeXp1QUcwdAppYzlqbjB3WlNrMWRua0dLVWxKQW5YSmd2OVdTYktWanZ6bXk4NDhTS0l3ZlhQUGd3K3RTVXBlWGx3VXJTQ3EwdVJtc0V6dzFuR3BKCnRuWjBLU2hYTGhFcFkwcERIN3NTczNmdkJONVBVYWlEUTJob00xR2JNMGxsa0syZEs4cjdpLzhkU0ZoTDRpMFJUQ0FRZUs5TUl3ZFoKV1ZrSGh4ZTRWN0hqYkRpRHJCTWJNMFZQUExERzhrb1plZ3RsZ2kwelc1NDhHVWF5ZVhsNW1XQmxaVitFQW8yZ2t2dDZwenlaZzZ5bApvZVhQQk43ZU16TlBJRTluVWVaZW5FelFtSWRLT1UyU0t3cVROM3FsZ2xyNTUrbjA5UFNINGVIaDJZOW85eUdjM1RoNzRiYzN6MEhLClRmQmhkdllIeUVjaS94RkJoM1lJODhWNkpzeTZMS0tSN2VycTZqZkNmQzBNMm5FS01qRXhZV1ptdG12WDZTNDFTUTB1NVpqdENWZjMKMzc3Nkh2TDVwNkJUUWdHL3EycHFXeVd2aFdoSjNMelZ2bDFkODhTZUwxMWQzZDNoRDBTUTJXbm82YnI2RXF0OUdoWnQzU1lVMDl4ego0a2kxU1hNT0xPQng5M0h3YUZjWE5LazkyNHBaVUJpMzIvL0c5ZUF6MFZrUGtHM2FIVXozWENMd09PcFN3MTJTKzRDRlpIQzF4aFFvCllzRjNzK1dQR0tCcUgzVVp6WXYxdHk4UkZyM2NpbG10UnBCRldGdVUxaWFKaGdDanhwMzZ4bG5pZmtoWFZWMUdSbk91L3NSdDZFWFAKQ0hiTklpUkRybVhiSmhOREN1Znd0enYxUitYYkNaa0dwWm9YNTRBaXlWcGd1STZMblVKNmdCaUdWNHl6NmtSclVKV3RUMW9UdEY2YwpxMGVTbUlWa3RIRzF0bTN5bDNoRjRSaVpBc3dlbGRlcmUrQUhFa29KaW1xQmhXWElSV0ZuWXBndWhMbEhvNDBEMjhVZnFLQXFYUjhmCjlhWTdNakpJd3lJTTZxSzA5cHVQbVNnMFVBTEVSakRVanphVzF6c2lhcUNDdHV1cStrRHBIUlFVRXRZbTF3THVPeE1KdzhzQm5Kc2oKd2I0QXJ4dzdGOWhlaDJXVnJxMnFqN282Q21tVDQ3YkFMbllOR0I0MjlUMlllelI3MUZnKzY0aTRxRUVOZElKTVEyRlJYSzN1L2VidwpRSVViNlpUckhLUER3YjRIeitqL0NOdUJlaUQ5WUwxS1YxVVZaUkRPM0I4WFVnSTRZaVByRUxibHM5ckZSUWxaWFcyTDJyakNQZ2xoCklTeGplSXZZSmlvTmFyRFVSVnk0QXg1b29nQXZwRndYTXhveGJmUTllQmRnTks3VXEvc2tVUXU0TVhpZ0lSUUdjT0FyQmpjK3g1RFkKRHRRakpGNUgrRXVQNFJlK1NyOU00WWdaZ2ZSOXZ2Tk1McTRVam1OWm8vSS9WOU4reDVDWjFhOEFBQUFBU1VWT1JLNUNZSUk9Ii8+PC9nPjwvc3ZnPg==);
```

Comment:
Watch out, this is a breaking change. Be sure to ask people if they were using the previous small version. 

If it was just me, i would prefer to keep the `s` one intact and created a `xs` version because there is more chance that we create, in a near futur, a new one between the large and previous small ones than creating one smaller than the new `s` one.

Reply:
xs, s and l is a very bad thing to do. Let's take advantage of the beginning of the ui-kit to correct this NOW and not later when it will be hard to turn back.